ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1093 @@
+
+Before starting to implement the QS on OpenShift, review <<_running_on_kubernetes, Running on Kubernetes>>.
+
+==== Running on OpenShift: Prerequisites
+
+include::../shared-doc/cd-create-project.adoc[leveloffset=+4]
+
+[#_running_on_openshift_start_postgresql_database]
+===== Running on OpenShift: Start PostgreSQL database
+
+For testing purposes you can use
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+# change path to ${PATH_TO_QUICKSTART_DIR}
+cd ${PATH_TO_QUICKSTART_DIR}
+# deploy not-production ready PostgreSQL database to OpenShift
+oc create -f ejb-txn-remote-call/client/extensions/postgresql.deployment.yaml
+----
+
+[#_running_on_openshift_build_the_application]
+==== Running on OpenShift: Build the application
+
+OpenShift provides the s2i functionality out-of-the-box.
+The quickstart defines two OpenShift templates.
+ The `BuildConfig` template builds the application based on the images
+configured in the `ImageStream` template. The `BuildConfig` template is named `wildfly-s2i`.
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}
+
+# Define the image streams of app server images.
+oc create -f ejb-txn-remote-call/openshift-image-streams.json
+# Upload the template definition for the builds named wildfly-s2i.
+oc create -f ejb-txn-remote-call/openshift-chain-build-config.json
+
+# Invoke the build via s2i chain build.
+oc new-app --template=wildfly-s2i -p IMAGE_STREAM_NAMESPACE=$(oc project -q) \
+  -p APPLICATION_NAME=client -p SOURCE_REPOSITORY_URL=https://github.com/wildfly/quickstart.git \
+  -p SOURCE_REPOSITORY_REF=master -p CONTEXT_DIR=ejb-txn-remote-call/client
+oc new-app --template=wildfly-s2i -p IMAGE_STREAM_NAMESPACE=$(oc project -q) \
+  -p APPLICATION_NAME=server -p SOURCE_REPOSITORY_URL=https://github.com/wildfly/quickstart.git \
+  -p SOURCE_REPOSITORY_REF=master -p CONTEXT_DIR=ejb-txn-remote-call/server
+----
+
+These commands create the `ImageStream` image, which contains the docker image
+of the {productName} runtime image with the deployed applications of this quickstart.
+
+Wait for the builds to finish. After a few minutes you can
+verify the build status by running the `oc get pod` command. The output should
+show that the `*-build` jobs have the value `Completed` in the `STATUS` column.
+
+[source,sh,options="nowrap"]
+----
+oc get pod
+NAME                                READY   STATUS      RESTARTS   AGE
+client-2-build                      0/1     Completed   0          35m
+client-build-artifacts-1-build      0/1     Completed   0          45m
+server-2-build                      0/1     Completed   0          15m
+server-build-artifacts-1-build      0/1     Completed   0          19m
+----
+
+[#_running_on_openshift_install_productname_operator]
+==== Running on OpenShift: Install {productName} Operator
+
+Installation of the {productName} Operator on OpenShift is summarized in
```

Comment:
I suggest you to drop all wrt Operator, again your Quickstart is quite complex and you should just stick to standard baremetal and Openshift instructions to simplify it as much as possible. As it is now the user easily looses focus from what your quickstart is really about.